doc/api_methods.md

```diff
@@ -114,20 +130,62 @@ An overlay is internally a [Backbone.View](http://backbonejs.org/#View) so if yo
 
 ### vis.addInfowindow(_map, layer, fields [, options]_)
 
-Adds an infowindow to the map controlled by layer events. It enables interaction and overrides the layer interactivity.
+Infowindows provide additional interactivity for your published map, controlled by layer events. It enables interaction and overrides the layer interactivity. A pop-up information window appears when a viewer clicks, or hovers their mouse over, select data on your map. 
+
+**Note:** By default, the `vis.addInfowindow` code displays interactivity for the infowindow with the "click" action. If you are using the "hover" action, you will still have to define the infowindow options with `viz.addInfowindow`, and specify additional parameters for the hover action with the `vis.Overlays` code.
 
 #### Arguments
 
 Option | Description
 --- | ---
-map | native map object or leaflet
-layer | cartodb layer (or sublayer)
-fields | array of column names
+map | native map object or leaflet.
```

Comment:
This method accepts 4 arguments: `map`, `layer`, `fields`, and `options`. `options` is the only optional argument. 

The only options I'd document are:
-  `options.infowindowTemplate` which allows users to set the HTML of the template. 
- `options.templateType` which sets the type of template (`underscore` or `mustache`).
- `options.triggerEvent` which can be one of `click` or `hover`.